CHANGES.rst

```diff
@@ -1,3 +1,11 @@
+Upcoming release
+================
+
+Changed features
+----------------
+- :code:`Instrument.control` does not apply :code:`get_process` to a returned list anymore. Use :code:`get_process_list` parameter instead.
```

Comment:
Maybe specify that get_process_list is applied only to returned list and that get_process is still applied when a single item is returned.

Reply:
I changed it.